systems/plants/@RigidBodyManipulator/addRobotFromSDF.m

```diff
@@ -312,6 +328,9 @@
         if (options.visual || options.visual_geometry)
          geometry = RigidBodyGeometry.parseSDFNode(geomnode,xyz,rpy,model,body.robotnum,options);
          if ~isempty(geometry)
+           if isa(geometry,'RigidBodyMesh')
+             geometry.scale=.0254; % gazebo dae meshes appear to be in inches
+           end
```

Comment:
Note: this could lead to confusion if someone tries loading a mesh that uses meters